spec/schema.rb

```diff
@@ -1,14 +1,17 @@
 ActiveRecord::Schema.define :version => 0 do
   create_table :flaggable_models, :force => true do |t|
     t.string :name
+    t.integer :flaggings_count
```

Comment:
What did you do in order to make these changes? Was it just running a `db:migrate`?